src/__stories__/menu-story.tsx

```diff
@@ -55,50 +72,50 @@ export const Default: StoryComponent<MenuArgs> = ({
                 >
                     <Menu
                         position={horizontalPosition}
-                        width={320}
+                        width={280}
                         renderTarget={({ref, onPress, isMenuOpen}) => (
                             <Touchable
                                 ref={ref}
```

Comment:
```suggestion
                                ref={ref}
                                aria-expanded={isMenuOpen}
```

Reply:
This has been fixed internally in Menu component